web-registry/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -31,6 +32,10 @@ const RegistryLayoutHeader: React.FC = () => {
   const headerColors = useMemo(() => getHeaderColors(theme), [theme]);
   const isTransparent = useMemo(() => getIsTransparent(pathname), [pathname]);
   const menuItems = useMemo(() => getMenuItems(pathname), [pathname]);
+  const userMenuItems = useMemo(
```

Comment:
UserMenu config is now defined in registry instead of web-components.

Reply:
nice!